lib/widgets/subscription_list.dart

```diff
@@ -124,11 +123,12 @@ class _NoSubscriptionsItem extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final designVariables = DesignVariables.of(context);
+    final zulipLocalizations = ZulipLocalizations.of(context);
 
     return SliverToBoxAdapter(
       child: Padding(
         padding: const EdgeInsets.all(10),
-        child: Text("No channels found",
+        child: Text(zulipLocalizations.noSubscriptions,
```

Comment:
This feels like a mismatch between the name and the actual string. That risks being confusing when looking at the code and thinking about how it relates to the UI. Let's try `subscriptionListNoChannels`.

That name starts by scoping where this is found, and then gives the gist of the string's actual content.